style.css

```diff
@@ -16,14 +16,19 @@ img {
 .nav {
     position: relative;
     display: flex;
-    width: 58px;
-    height: 11px;
-    left: 811px;
-    top: 29px;
-    color: blanchedalmond;
+    align-items: center;
+    text-align: right;
+    color: #ffffff;
     flex-direction: row;
+    flex-flow: row;
+    width: 100%;
+    justify-content: flex-end;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
 }
 
 .nav_link {
-    list-style-type: none;
-}
\ No newline at end of file
+    margin-right: 50px;
```

Comment:
As you can see the text is not centered inside the button, this is bad if we for example want to add a hover animation.

Please center the text horizontally inside a nav-item.

![image](https://user-images.githubusercontent.com/13898415/144058201-e5c81998-baca-4aed-a619-4f5c22100b3a.png)